R/pkg/R/context.R

```diff
@@ -225,3 +225,17 @@ broadcast <- function(sc, object) {
 setCheckpointDir <- function(sc, dirName) {
   invisible(callJMethod(sc, "setCheckpointDir", suppressWarnings(normalizePath(dirName))))
 }
+
+#' Set new log level
+#'
+#' Set new log level: "ALL", "DEBUG", "ERROR", "FATAL", "INFO", "OFF", "TRACE", "WARN"
+#' @param sc Spark Context to use
+#' @param level New log level
+#' @examples
```

Comment:
please add `@rdname` and `@export` like [here](https://github.com/apache/spark/commit/14869ae64eb27830179d4954a5dc3e0a1e1330b4#diff-d97f9adc2dcac0703568c799ff106987R1018)